README.md

```diff
@@ -102,7 +103,7 @@ WP Rig gives the developer an out of the box environment with support for modern
 
 Configuring the behavior of WP Rig is done by editing `./config/config.json`. Here the developer can set the theme name and theme author name (for translation files), and local server settings for BrowserSync. Additionally, compression of JavaScript and CSS files can be turned off for debugging purposes.
 
-Place your custom theme settings in `./config/config.json` to override default settings, located in `./config/config.default.json`.
+Place your custom theme settings in `./config/config.json` to override default settings, located in `./config/config.default.json`. Place local-only theme settings in `./config/config.local.json`, e.g. potentially sensitive info like the path to your BrowserSync certificate.
```

Comment:
@bamadesigner can you expand this a bit? I would like to see the namespace options added as an example of what should go in `config.json`